rust/kcl-lib/src/execution/geometry.rs

```diff
@@ -561,6 +561,13 @@ impl Plane {
     pub fn is_standard(&self) -> bool {
         !matches!(self.value, PlaneType::Custom | PlaneType::Uninit)
     }
+
+    pub fn project(&self, point: Point3d) -> Point3d {
```

Comment:
Could you please add a little comment for those among us who don't properly understand geometry well :)